plugin/src/integrationTest/kotlin/com/vanniktech/maven/publish/ProjectSpecRunner.kt

```diff
@@ -41,8 +41,8 @@ fun ProjectSpec.run(fixtures: Path, temp: Path, options: TestOptions): ProjectRe
 }
 
 private fun supportsConfigCaching(plugins: List<PluginSpec>): Boolean {
-  // TODO https://github.com/Kotlin/dokka/issues/2231
-  return !plugins.any { it.id == dokkaPlugin.id }
+  // TODO can always return true when dropping support for dokka 1
+  return plugins.none { it.id == dokkaPlugin.id }
```

Comment:
Config cache doesn't like the `provider.flatMap { tasks.named(it) }` that we're doing right now. So until we're dropping support for Dokka 1 it will stay incompatible.

Reply:
FYI: Configuration cache is supported ONLY when pluginMode is `V2Enabled/V2EnabledWithHelpers` - so in so called DGP v2

Reply:
👍 I've clarified/updated the comment